lib/config/recommended.js

```diff
@@ -2,31 +2,34 @@
 
 module.exports = {
   rules: {
-    'block-indentation': true,
     'deprecated-render-helper': true,
-    'img-alt-attributes': true,
-    'linebreak-style': true,
-    'link-rel-noopener': true,
+    'link-rel-noopener': 'strict',
+    'link-href-attributes': true,
+    'no-abstract-roles': true,
     'no-attrs-in-components': true,
     'no-debugger': true,
     'no-duplicate-attributes': true,
+    'no-extra-mut-helper-argument': true,
     'no-html-comments': true,
+    'no-implicit-this': true,
```

Comment:
are we sure we want to enable this by default?

I was under the impression that we only wanted to enable this for the `octane` preset for now. If we do enable this, then why not also the `no-curly...` rule? 🤔

Reply:
@Turbo87 - Ya, agree. I'll update.

Reply:
Taken care of in #1045